### ISSUE TITLE
nightly/dev/canary channel

### ISSUE THREAD

**Issue by satra** [conda-forge/staged-recipes#2113]
has there been any consideration of a nightly channel or a dev conda channel that builds the master (or specified other) branch of a project on a nightly or other frequent basis that is different from official releases?

**Comment by jakirkham** [comment]
Yeah I proposed this to others some time ago. I think it feel off the radar. Would definitely be interested to get people's feedback on this again.

cc @conda-forge/core

**Comment by hmaarrfk** [comment]
I would definitely be interested in this.

With CIs, it is much easier to catch breaking bugs in code if you have a good test suite.

Either nightly, or the ability to push to a custom channel sound like acceptable ideas to me. 
The idea of custom channels seems to have been discussed in #593, but that seems to have moved toward: "mirroring certain packages on the conda-forge system".
